cypress/support/pages/homeCatalogue.pageObject.js

```diff
@@ -17,6 +17,89 @@ class HomeAndCataloguePageObject extends PageObject {
     cy.contains('.hrefch', product)
       .click();
   }
+
+  clickOnAddToCartBtn() {
+    cy.contains('.btn', 'Add to cart')
+      .click();
+  }
+
+  verifyAddingToCart() {
+    return this.assertAllert('Product added');
```

Comment:
The method `assertAllert` seems to be a typo or undefined in the context of this class. Ensure that the correct method name is used or that it is properly defined elsewhere.